docfx_yaml/extension.py

```diff
@@ -400,6 +400,7 @@ def _parse_docstring_summary(summary):
     summary_parts = []
     attributes = []
     attribute_type_token = ":type:"
+    enum_type_token = "Values:"
```

Comment:
If it was just `values` docstring tokenized I would have kept it up there but since we're looking for the specific `Values:` string I've kept it here.